packages/bff/src/services/clientService.ts

```diff
@@ -331,14 +331,14 @@ async function getSelfcareUserById(
   selfcareClient: SelfcareV2UsersClient,
   userId: string,
   selfcareId: string
-) {
+): Promise<selfcareV2ClientApi.UserResponse> {
   try {
     return selfcareClient.getUserInfoUsingGET({
       params: { id: userId },
       queries: { institutionId: selfcareId },
     });
   } catch (error) {
-    throw userNotFound(userId, selfcareId);
+    return {};
```

Comment:
What's the reason behind not throwing an error here? 
Btw the error is logged in an interceptor created specifically for the selcareClient ([here](https://github.com/pagopa/interop-be-monorepo/blob/main/packages/api-clients/src/selfcareClients.ts))

Reply:
Because of this: https://github.com/pagopa/interop-be-monorepo/pull/753#discussion_r1688334840
In scala, apparently no error is thrown, instaed an empty user object is returned. This of course changes the behaviour. My doubt is whether to return empty user on any error or only on specific errors.